EIPS/eip-2942.md

```diff
@@ -0,0 +1,65 @@
+---
+eip: 2942
+title: EthPM URI Specification
+author: Nick Gheorghita (@njgheorghita), Piper Merriam (@pipermerriam), g. nicholas d'andrea (@gnidan), Benjamin Hauser (@iamdefinitelyahuman)
+status: Draft
```

Comment:
Missing discussion-to which is required for merging.  Not sure why the bot didn't catch that nor I catch it previously.

Reply:
I've made the update, thanks!